lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,7 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <img src="/static/images/xseries-certificate-visual.png" alt="XSeries Program Certificates" aria-hidden="true">
```

Comment:
@AlasdairSwan I have seen this in codebase [static_link](https://github.com/edx/edx-platform/blob/master/lms/static/js/verify_student/views/webcam_photo_view.js#L159)

@schenedx gave a good idea I can pass the image url from template.